### ISSUE TITLE
[Crash] Blueprint with a lot of screens causes crash on load

### ISSUE THREAD

**Issue by Alkaliii** [tsunamayo/Starship-EVO#5425]
### Short Description of the issue:
# I MODIFIED THE BLUEPRINT OUTSIDE OF THE GAME
Hello, I would like to preface this with a warning, this crash probably happened because the blueprint was **modified outside the game**, not with EvoEdit but with a program I wrote myself. I have a blueprint with 900 screens, each filled to the brim with text. When I select the blueprint to load in my game freezes for a bit but if I get lucky it frees up, if I don't it crashes. In the event it does load, only some of the screens display properly, and the rest show no text. I think it's related to ram on my system, but I thought you might be interested in the modified BP since it sorta stress tests screen blocks.

Sorry for taking up your time!

TLDR: Loading the blueprint either causes a crash or loads the bp with some of the screens not displaying anything.
...

### Steps to reproduce the issue:
- Load BP
- Crash
...

### Environment
- Starship EVO version: 24w02a
- Graphics Card: NVIDIA GeForce RTX 3060

### Additional information:

result (pink screens are intended)
![image](https://github.com/tsunamayo/Starship-EVO/assets/53021785/fc065afb-116b-48e9-bdb0-078220522395)

[expected] result after reloading from desktop.
![image](https://github.com/tsunamayo/Starship-EVO/assets/53021785/29e531fb-84dd-4c30-ae0c-69fc8c19c2c6)

Please also include the `Player.log` file. You can find the log file here: `%UserProfile%\AppData\LocalLow\Moonfire Entertainment\Starship EVO\`

[Player.log](https://github.com/tsunamayo/Starship-EVO/files/13926601/Player.log)
[Player-prev.log](https://github.com/tsunamayo/Starship-EVO/files/13926602/Player-prev.log)

Blueprints (both were modified outside the game):

this one has 1 screen.
[testGodot.zip](https://github.com/tsunamayo/Starship-EVO/files/13926612/testGodot.zip)

this one has 900.
[testGodotB.zip](https://github.com/tsunamayo/Starship-EVO/files/13926613/testGodotB.zip)

**Comment by CoasterBlock** [comment]
I also experienced the game lagging out/freezing when pasting in the text-data for the screen. Copy/paste didnt cause trouble, the game stutters and freezes only when adding a lot of text to the screen.

**Comment by tsunamayo** [comment]
@Alkaliii Hi, thanks for the bp, yes indeed screen are not optimized at the moment. So each new screen add its burden to the rendering engine, dont use too much for now. Cheers
